lib/storage.js

```diff
@@ -23,6 +23,11 @@ bedrock.events.on('bedrock-mongodb.ready', async () => {
       background: false,
       expireAfterSeconds: 0
     }
+  }, {
+  // index for query that is used to look up tokens
+    collection: 'oauth2-client-accessToken',
+    fields: {'token.client_id': 1, 'token.token_endpoint': 1},
```

Comment:
Now that we have this index, do we need the `token.token` unique index? It seems like it could even technically be "ok" if two tokens with the same value were stored if they were for a different client + endpoint (even though if this happened it would likely be an indication of some problem with the authz servers).

Reply:
cc: @dmitrizagidulin

Reply:
Agreed, we do not need that index to be unique.

Reply:
Updated.